documentation/popups.md

```diff
@@ -193,3 +202,9 @@ data:
         }
 ```
 Or through `card-mod-more-info` or `card-mod-more-info-yaml` in a card-mod theme.
+
+## Nested popups
```

Comment:
Can this be permanently enabled, or would that mess something else up?

How does this work with the browser history? Some popups can be closed and reopened by using the browsers back-navagation. Having them nested may lead to surprising results.

Reply:
> Can this be permanently enabled, or would that mess something else up?
> 

There could be a chance someone likes the more-info cancelling the popup? Perhaps just make the option default to True rather than False? Also happy to rework code and test to make it just standard.

> How does this work with the browser history? Some popups can be closed and reopened by using the browsers back-navagation. Having them nested may lead to surprising results.

Tested and seems to work fine with back button.

Reply:
I have tested standard more-info going back and forward, and I am not seeing the standard more-info reopen. Anyhow, as you say we can leave for now. 

I have now pushed a commit to make the option default to true.